packages/hoppscotch-common/src/components/app/Header.vue

```diff
@@ -250,20 +250,44 @@
       @dismiss="dismissBanner"
     />
     <TeamsModal :show="showTeamsModal" @hide-modal="showTeamsModal = false" />
-    <TeamsInvite
-      v-if="workspace.type === 'team' && workspace.teamID"
-      :show="showModalInvite"
-      :editing-team-i-d="editingTeamID"
-      @hide-modal="displayModalInvite(false)"
-    />
-    <TeamsEdit
-      :show="showModalEdit"
-      :editing-team="editingTeamName"
-      :editing-team-i-d="editingTeamID"
-      @hide-modal="displayModalEdit(false)"
-      @invite-team="inviteTeam(editingTeamName, editingTeamID)"
-      @refetch-teams="refetchTeams"
-    />
+    <template v-if="platform.ui?.teamInviteComponent">
+      <component
+        :is="platform.ui.teamInviteComponent"
+        :show="showModalInvite"
+        :editing-team-i-d="editingTeamID"
+        @hide-modal="displayModalInvite(false)"
+      />
+    </template>
+    <template v-else>
+      <TeamsInvite
+        v-if="workspace.type === 'team' && workspace.teamID"
+        :show="showModalInvite"
+        :editing-team-i-d="editingTeamID"
+        @hide-modal="displayModalInvite(false)"
+      />
+    </template>
```

Comment:
Let's remove the extra `<template>` wrappers when they contain only a single element. We can directly apply the directive (`v-if`) to the component. Applies throughout.